src/app/components/AllOrNone.tsx

```diff
@@ -10,15 +12,19 @@ interface Props {
 }
 
 // tslint:disable-next-line:variable-name
-const AllOrNone = ({className, onAll, onNone}: Props) => <div className={className}>
-  <FormattedMessage id='i18n:highlighting:filters:all'>
-    {(msg: Element | string) => <ButtonLink decorated onClick={onAll}>{msg}</ButtonLink>}
-  </FormattedMessage>
-  <span>|</span>
-  <FormattedMessage id='i18n:highlighting:filters:none'>
-    {(msg: Element | string) => <ButtonLink decorated onClick={onNone}>{msg}</ButtonLink>}
-  </FormattedMessage>
-</div>;
+const AllOrNone = ({className, onAll, onNone}: Props) => {
+  const userLoggedOut = useSelector(loggedOut);
```

Comment:
this component is shared in places outside the scope of what you're trying to change, you can't add this logic here.

you could add a `disabled` prop, that would be ok